index.html

```diff
@@ -2127,11 +2127,16 @@ <h2>Definition of Roles</h2>
           If the <code>combobox</code> popup element has a role other than <rref>listbox</rref>, authors MUST specify a value for <pref>aria-haspopup</pref> that corresponds to the role of its popup.
         </p>
 				<p>
-          If the user interface includes an additional icon that allows the visibility of the popup to be controlled via pointer and touch events, authors SHOULD ensure that element has role <rref>button</rref> and that it is focusable but not included in the page <kbd>Tab</kbd> sequence.
+          If the user interface includes an additional icon that allows the visibility of the popup to be controlled via pointer and touch events, authors SHOULD ensure that element has role <rref>button</rref>, that it is focusable but not included in the page <kbd>Tab</kbd> sequence, and that it is not a descendant of the element with role <code>combobox</code>.
           In addition, to be keyboard accessible, authors SHOULD provide keyboard mechanisms for moving focus between the <code>combobox</code> element and elements contained in the popup.
           For example, one common convention is that <kbd>Down Arrow</kbd> moves focus from the input to the first focusable descendant of the popup element.
           If the popup element supports <pref>aria-activedescendant</pref>, in lieu of moving focus, such keyboard mechanisms can control the value of <pref>aria-activedescendant</pref> on the <code>combobox</code> element.
           When a descendant of the popup element is active, authors MAY set <pref>aria-activedescendant</pref> on the <code>combobox</code> to a value that refers to the active element within the popup while focus remains on the <code>combobox</code> element.
+        </p>
+        <p>
+          User agents MUST compute a value for elements with role <code>combobox</code> and expose it to <a>assistive technologies</a>.
+          If the <code>combobox</code> element is also a host language element that provides a value, such as an HTML <code>input</code> element, the value of the combobox is the value of that element.
+          Otherwise, the value of the <code>combobox</code> is computed from descendant elements of the <code>combobox</code> using the same algorithm that is used to compute the name of a <rref>button</rref> from its descendant content.
```

Comment:
> using the same algorithm that is used to compute the name of a button

I can live with this change as-is, but are we postponing a probleme by not adding a "value computation" to ACC-NAME?

Reply:
Since accname 1.2 is on an independent, later  time table, we need to write this based on accname 1.1.

I think we can draft a value calculation in accname 1.2 and then revise this wording in ARIA 1.3 after accname 1.2 is final.

We could base the accname 1.2 draft on what we would also need if we want to revise aria-valuetext to have a broader scope and get closer to attribute parity with HTML value. Such changes to accname would also provide the support needed for an aria-valuedby attribute if we decide we want that as well.

Reply:
@mcking65 Reminder that on the call today, @joanmarie stated a preference for not using the word compute or computed, and instead using something more along the lines of... "exposed using the same method [as for determining the label of a button]" or similar.